xbmc/windowing/gbm/DRMUtils.cpp

```diff
@@ -582,25 +582,29 @@ void CDRMUtils::DestroyDrm()
   m_fd = -1;
 
   drmModeFreeResources(m_drm_resources);
-  m_drm_resources = nullptr;
 
+  drmModeFreeConnector(m_connector->connector);
+  drmModeFreeObjectProperties(m_connector->props);
+  delete [] m_connector->props_info;
   delete m_connector;
-  m_connector = nullptr;
```

Comment:
why is this removed? Could make sense if you call this methode twice (by failure), because it will lead to a segfault.